gazebo/gazebo8/libgazebo8/Dockerfile

```diff
@@ -1,8 +1,14 @@
 # This is an auto generated Dockerfile for gazebo:libgazebo8
 # generated from templates/docker_images/create_gzclient_image.Dockerfile.em
-# generated on 2017-01-26 20:10:11 +0000
+# generated on 2017-01-26 20:31:00 +0000
 FROM gazebo:gzserver8
 MAINTAINER Jose Luis Rivero jrivero@osrfoundation.org
```

Comment:
There is no problem for keeping Steve as the maintainer I just put my name since I was creating the image. If we need to keep just one contact, we can use my name.